app/src/main/java/app/musikus/ui/library/LibraryScreen.kt

```diff
@@ -440,7 +446,6 @@ fun LibraryContent(
                 ) {
                     LibraryUiItem(
                         modifier = Modifier.padding(
-                            vertical = MaterialTheme.spacing.small,
```

Comment:
I noticed that the paddings around the library items are pretty large. Is this intentional?

Reply:
Ok, I figured as much. Let's review the Library screen together. Or in another PR?

Reply:
I removed the additional horizontal padding there was from before and now they look normal